.github/workflows/test.yml

```diff
@@ -13,3 +13,11 @@ jobs:
     - uses: actions/checkout@v2
     - name: Dummy command
       run: touch foo 
+#    - name: First one fails # this one does fail and causes the next run to be skipped!
+#      run: |
+#        false
+#        true
+#     - name: Seconnd one fails # this one also fails
```

Comment:
Say, turns out this line is also wrong but I didnt see it during the first round of reviews...